trio/_subprocess_platform/waitid.py

```diff
@@ -101,7 +101,7 @@ async def wait_child_exiting(process: "_subprocess.Process") -> None:
     #   process.
 
     if process._wait_for_exit_data is None:
-        process._wait_for_exit_data = event = Event()  # type: ignore
```

Comment:
restrict the `type: ignore`

Reply:
This is actually removing the type-ignore, it’s no longer required.

Reply:
oh derp